metashare/api/models.py

```diff
@@ -153,6 +154,8 @@ def sf_username(self):
         try:
             return self.salesforce_account.extra_data["preferred_username"]
         except (AttributeError, KeyError):
+            if settings.DEVHUB_USERNAME:
+                return settings.DEVHUB_USERNAME
             return None
 
     @property
```

Comment:
@wlonk Below in `is_devhub_enabled` (https://github.com/oddbird/MetaShare/pull/202/files#diff-c560aacecf55d18e2e99a8b7f0190423R186-R194), I think we also want to `return True` `if settings.DEVHUB_USERNAME and not self.devhub_username and not self.allow_devhub_override`.

My understanding of the logic is:
- If `self.devhub_username`, use that.
- Else if `settings.DEVHUB_USERNAME and not self.allow_devhub_override`, use `settings.DEVHUB_USERNAME`
- Else use `self.salesforce_account`